src/plugins/query_enhancements/server/plugin.ts

```diff
@@ -103,6 +104,7 @@ export class QueryEnhancementsPlugin
     this.logger.info('queryEnhancements: Setup complete');
     return {
       defineSearchStrategyRoute: defineSearchStrategyRouteProvider(this.logger, router),
+      prometheusManager: new PrometheusManager(),
```

Comment:
not needed right now but it can be convenient to add `prometheusManager` to route handler context, since query enhancement will also define routes that will eventually get resources using `prometheusManager`

Reply:
Gotcha. I'll make sure to include this in an upcoming PR.